fs/dafsclient/rmtfile.cpp

```diff
@@ -616,13 +616,13 @@ class CCritTable : private SimpleHashTableOf<CEndpointCS, const SocketEndpoint>
     CEndpointCS *getCrit(const SocketEndpoint &ep)
     {
         CriticalBlock b(crit);
-        Linked<CEndpointCS> clientCrit = find(ep);
-        if (!clientCrit || !clientCrit->isAlive()) // if !isAlive(), then it is in the process of being destroyed/removed.
+        CEndpointCS * clientCrit = find(ep);
+        if (!clientCrit || !clientCrit->isAliveAndLink()) // if !isAlive(), then it is in the process of being destroyed/removed.
```

Comment:
commend should be updated: // if !isAliveAndLink(),

Reply:
done